provider/lxd/environ_broker.go

```diff
@@ -179,7 +179,7 @@ func (env *environ) getContainerSpec(
 	}
 	cSpec := lxd.ContainerSpec{
 		Name:     hostname,
-		Profiles: []string{"default", env.profileName()},
+		Profiles: append([]string{"default", env.profileName()}, args.CharmLXDProfiles...),
```

Comment:
I was wondering if order matters?

Reply:
"Profiles are applied in the order they are specified so the last profile to specify a specific key wins."  https://lxd.readthedocs.io/en/latest/profiles/